Microsoft/WebTemplateStudio
# Add Master Detail Merging and Routing to Flask Templates

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/27728

This PR adds master detail Routing support using Sample Data to the Flask Application.

**How to Check**
- Generate a Flask Application
- Go the application directory and run yarn install
- Run 'yarn start` to run the application
- Go to /Master_Detail route to see if it is working

Closes #724
